extensions/exposition/features/steps/components/octets.tester/operations/yex.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+import { setTimeout } from 'node:timers/promises'
+
+async function * effect (_) {
+  await setTimeout(10)
+  yield 'hello'
+
+  await setTimeout(10)
+  yield 'world'
+
+  await setTimeout(10)
+  throw new Error('Oops!')
+}
```

Comment:
We are repeating this logic twice (in `yield.js` also) - so we may benefit from LatencyStream class or something